dspace/config/emails/subscription

```diff
@@ -0,0 +1,12 @@
+## E-mail sent to DSpace users when new items appear in collections they are
```

Comment:
This email template filename should be `subscription_content` as appears to be used [here](https://github.com/TAMULib/DSpace/blob/tamu-dspace-7_x/dspace-api/src/main/java/org/dspace/subscriptions/ContentGenerator.java#L57).